Makefile

```diff
@@ -232,6 +232,11 @@ lint-metrics:
 	./hack/prom-metric-linter/metric_name_linter.sh --operator-name="kubevirt" --sub-operator-name="kubevirt" --metrics-file=metrics.json
 	rm metrics.json
 
+update-generate-api-testdata:
```

Comment:
can we also add `test-api-compat` that will invoke these tests without `UPDATE_COMPATIBILITY_FIXTURE_DATA=true` for all three previous versions

Reply:
Done added

Reply:
Whenever there is a change in the schema, this make command helps in adding or deleting those fields in the HEAD testdata. Hence, used the update prefix.